### PR TITLE
Fix/tao 7637 droppable.js placeholder removed from bottom

### PR DESCRIPTION
**Pull request by rkarolis**
https://oat-sa.atlassian.net/browse/TAO-7637

In item authoring, by dragging interaction the placeholder on the item page should be removed by going through bottom.
Minor refactoring.

### CODE REVIEW

File: scripts/update/Updater.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  *
- * Copyright (c) 2016 (original work) Open Assessment Technologies SA;
+ * Copyright (c) 2014-2019 (original work) Open Assessment Technologies SA;
```
Comment (ferenckiss89) [inline scripts/update/Updater.php]: Why 2014? it should be either 2016-2019 if that's the original date or simply 2019 with the latest changes.

File: views/js/qtiCreator/editor/gridEditor/droppable.js [hunk 2]
```diff
@@ -305,8 +305,10 @@ define([
         ).on('mouseleave.gridEdit.gridDragDrop',
             _.debounce(function(e){
                 var goingTo = e.relatedTarget|| e.toElement; //browser compatibility
+                var $target = $(e.target);
 
-                if($(e.target).parents('.qti-itemBody').length) {
+                if($target.parents('.qti-itemBody').length ||
```
Comment (ferenckiss89) [inline views/js/qtiCreator/editor/gridEditor/droppable.js]: I'd put also $target && .. just to make sure but if we did not used to, then it's not necessary.

File: scripts/update/Updater.php (hunk 1 above)
Comment (rkarolis) [reply scripts/update/Updater.php]: because original code is dating from 2014

File: views/js/qtiCreator/editor/gridEditor/droppable.js (hunk 2 above)
Comment (rkarolis) [reply views/js/qtiCreator/editor/gridEditor/droppable.js]: No we dont do and over here

Comment (rkarolis) [reply views/js/qtiCreator/editor/gridEditor/droppable.js]: $target is always here, otherwise there would be no event

**Pull request merged**